src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-weight: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.container {
+  position: relative;
+}
+
+.search-icon {
+  position: absolute;
+}
+
+.icon1 {
```

Comment:
dont use 1 and 2 in your classnames, use words to describe classes, like small, big, etc.